packages/manager/modules/vps/src/dashboard/components/vps-announcement-banner/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,4 @@
+{
+  "pci_components_vps_announcement_banner_info": "Maintenance VPS avec impacte sur la management de votre service.",
```

Comment:
Are you sure about this content? Looks like a shorthand version.

Reply:
Yep, it's validated by reporter.

Reply:
```suggestion
  "pci_components_vps_announcement_banner_info": "Maintenance VPS avec impact sur la management de votre service.",
```